Modules/mathmodule.c

```diff
@@ -1032,7 +1040,13 @@ math_2(PyObject *const *args, Py_ssize_t nargs,
 
 #define FUNC1(funcname, func, can_overflow, docstring)                  \
     static PyObject * math_##funcname(PyObject *self, PyObject *args) { \
-        return math_1(args, func, can_overflow);                            \
+        return math_1(args, func, can_overflow, NULL);                  \
+    }\
+    PyDoc_STRVAR(math_##funcname##_doc, docstring);
+
+#define FUNC1D(funcname, func, can_overflow, docstring, err_msg)        \
```

Comment:
Could you perhaps add a comment about why there is a D suffix? we have math1 and func1 being matched but here we don't use a math1d function.

Reply:
I think this is not needed. Redundant trivial comments do not make the code better.

Reply:
Ok! Maybe I haven't followed the discussion enough but what was the reason behind the D suffix? was it for "domain"? (if so, I think I'm good with no comment)